collectors/ebpf.plugin/ebpf.c

```diff
@@ -1120,51 +726,17 @@ int main(int argc, char **argv)
 
     set_global_variables();
 
-    if (load_collector_file(user_config_dir)) {
+    if (load_collector_config(user_config_dir)) {
         info("[EBPF PROCESS] does not have a configuration file. It is starting with default options.");
```

Comment:
I don't like that there is manually added prefix in every log. You are setting the program_name + you should be able to use thread tags to get this automatically.

Reply:
```
typedef struct {
    void *arg;
    pthread_t *thread;
    const char *tag;
    void *(*start_routine) (void *);
    NETDATA_THREAD_OPTIONS options;
} NETDATA_THREAD;

static __thread NETDATA_THREAD *netdata_thread = NULL;

const char *netdata_thread_tag(void) {
    return ((netdata_thread && netdata_thread->tag && *netdata_thread->tag)?netdata_thread->tag:"MAIN");
}
```

Reply:
I agree, the first time this collector was created, it was only for process, now we have to update it.

Reply:
Yes best would be if we can already use the existing agent logging functionality which can already detect and print the thread name (if used in right way). Then you have logging solved also for individual threads that you add now/in the future.

Reply:
Done!